modules/social_features/social_profile/social_profile.install

```diff
@@ -481,3 +486,65 @@ function social_profile_update_8903() {
     ->set('enable_profile_tagging', TRUE)
     ->save();
 }
+
+/**
+ * Update the field definitions on install, or in an update hook.
+ */
+function _social_profile_field_definitions_update() {
+  // Create field storage for the 'Profile Name' base field.
+  \Drupal::entityTypeManager()->clearCachedDefinitions();
+  \Drupal::service('entity.definition_update_manager')->applyUpdates();
+}
+
+/**
+ * Install Profile name field to profile entity type.
+ */
+function social_profile_update_8904() {
+  $field_storage_definition = BaseFieldDefinition::create('string')
+    ->setLabel(t('Profile name'))
+    ->setCardinality(FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED)
+    ->setReadOnly(TRUE);
+
+  $update_manager = \Drupal::entityDefinitionUpdateManager();
+  $update_manager->installFieldStorageDefinition('profile_name', 'profile', 'social_profile', $field_storage_definition);
+}
+
+/**
+ * Install Profile name field to profile entity type.
+ */
+function social_profile_update_8905() {
+  // Update field definitions.
+  _social_profile_field_definitions_update();
+}
+
+/**
+ * Update Profile names.
+ */
+function social_profile_update_8906(&$sandbox) {
```

Comment:
Please move this code in a hook_post_update_name, as this has the potential to update thousands of profiles and may block hook_updates if kept failing.

Reply:
I have moved it.